modules/developers-guide/pages/cli-reference.adoc

```diff
@@ -631,18 +666,24 @@ However, the string label is only present for better readability.
 Behind the scenes, the string label is converted into a natural number using a fixed hash function. 
 If the canister method returns a `+record+` or `+variant+` type, the field labels will always be natural numbers.
 
-IMPORTANT: For this Alpha release, some features of the interface descriptions language are not yet supported.
+==== Limitations for arguments
 
-The following limitations apply for IDL arguments:
+For this release, the following limitations apply for IDL arguments:
 
 * No support for type annotations.
 * No support for Bignum.
 
-The following limitations apply for displaying method return values:
+==== Limitations for return values
+
+For this release, the following limitations apply for displaying method return values:
 
 * No support for Bignum.
 * All other IDL values are fully supported, including unicode text.
```

Comment:
All of these are supported, not more limitations!

Reply:
awesome!

Reply:
removed limitations.